### PR TITLE
[11.0][MIG] project_task_category: Migration to 11.0.

### PR DESCRIPTION
**Pull request by mamcode**

### CODE REVIEW

**Comment by ivantodorovich** [comment]
This is ok to pr to oca

The only thing I'd do is squashing the translation commits
https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

**Comment by ivantodorovich** [comment]
Also your commit should be "[MIG]" not "[ADD]"

File: project_task_category/migrations/11.0.1.0.0/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+                 (name, create_uid, create_date, write_uid, write_date)
+                SELECT name, create_uid, create_date, write_uid, write_date
+                FROM project_category_main
+                WHERE NOT EXISTS
+                 (SELECT id FROM project_category
+                  WHERE project_category_main.name = project_category.name
+                 );
+            """
+    openupgrade.logged_query(cr, query)
+    openupgrade.logging("Migrate project_category done.")
```
Comment (ivantodorovich) [inline project_task_category/migrations/11.0.1.0.0/post-migration.py]: The migrations folder should be removed
This is a migration script for a `10.0.x` version, it's not needed in `11.0`

> Remove any possible migration script from previous version.

is in the TODO list of https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
actually just foudn something

File: project_task_category/migrations/11.0.1.0.0/post-migration.py (hunk 1 above)
Comment (mamcode) [reply project_task_category/migrations/11.0.1.0.0/post-migration.py]: @ivantodorovich Please review again.

**Pull request closed without merging**